cloudformation/memsub-promotions-cf.yaml

```diff
@@ -400,6 +399,11 @@ Resources:
         FromPort: '9000'
         ToPort: '9000'
         SourceSecurityGroupId: !Ref 'LoadBalancerSecurityGroup'
+      SecurityGroupEgress:
```

Comment:
so in future the promo tool will only be able to make outbound connections on port 443, rather than the default of all outbound ports being allowed